
Oh my god is Gnome full of nice code duplications. This is exactly the same
patch as gnome-terminal needs (yes applies cleanls ;-): Some real bugs
uncovered by gcc-3.4 ...

  - Rene Rebe <rene@rocklinux.org>

--- gnome-terminal-2.6.0/src/eggcellrendererkeys.c.orig	2004-05-10 13:41:45.587354088 +0200
+++ gnome-terminal-2.6.0/src/eggcellrendererkeys.c	2004-05-10 13:42:16.827604848 +0200
@@ -584,7 +584,7 @@
 void
 egg_cell_renderer_keys_set_accelerator (EggCellRendererKeys *keys,
                                         guint                keyval,
-                                        GdkModifierType      mask)
+                                        EggVirtualModifierType mask)
 {
   char *text;
   gboolean changed;
@@ -626,7 +626,7 @@
 void
 egg_cell_renderer_keys_get_accelerator (EggCellRendererKeys *keys,
                                         guint               *keyval,
-                                        GdkModifierType     *mask)
+                                        EggVirtualModifierType *mask)
 {
   g_return_if_fail (EGG_IS_CELL_RENDERER_KEYS (keys));
 
